src/sass/components/_button.scss

```diff
@@ -12,6 +12,16 @@
     text-decoration: none;
   }
 }
+.btn-main-more {
+  background-color: #2a2a2a;
```

Comment:
Use SCSS variable to keep colors consistent.

Reply:
Variable added